.github/workflows/build.yml

```diff
@@ -92,6 +116,10 @@ jobs:
         run: |
           VERSION=`node -p "require('./package.json').devDependencies['semantic-release']"`
           echo "::set-output name=version::$VERSION"
+      - name: Download build
```

Comment:
If we only need the artifact to publish, I think that `action/checkout@v2` can be remove ?

Reply:
To be honest, I don't know. How is the release performed ? Will the action merge the whitelisted content from the github repo + the cached dist folder ? 

This is something I can test actually, can you remind me the steps to attempt a release on beta ?

Reply:
Done